doc/whats_new/v1.5.rst

```diff
@@ -158,6 +158,14 @@ Changelog
   prefix any feature names and will error if feature names are not unique.
   :pr:`25991` by :user:`Jiawei Zhang <jiawei-zhang-a>`.
 
+:mod:`sklearn.cluster`
+......................
+
+- |FIX| : Create copy of precomputed sparse matrix within the
```

Comment:
```suggestion
- |FIX| Create copy of precomputed sparse matrix within the
```

Reply:
By the way, can you please move this section such that the modules are sorted in alphabetical order?